#!/usr/bin/env python
PACKAGE = "robosense_pointcloud"

from math import pi
import dynamic_reconfigure.parameter_generator_catkin as pgc

gen = pgc.ParameterGenerator()

gen.add("view_direction", 
  pgc.double_t, 
  0, 
  "angle defining the center of view",
  0.0, -pi, pi)
  
gen.add("view_width", 
  pgc.double_t, 
  0, 
  "angle defining the view width",
  2*pi, 0.0, 2*pi)
  
gen.add("frame_id", 
  pgc.str_t, 
  0, 
  "fixed frame of reference for point clouds",
  "map")

exit(gen.generate(PACKAGE, "transform_node", "TransformNode"))
